save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -481,18 +479,16 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
     private fun postImageUpload(file: File) =
```

Comment:
btw, `postImageUpload` also looks similar for organization and user (only type is different)

Reply:
Indeed.

https://github.com/saveourtool/save-cloud/pull/1711#discussion_r1067947289

Reply:
Fixed